### PR TITLE
Refactoring/action menus

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/5509

### CODE REVIEW

**Comment by manwithtwowatches** [comment]
@tilt this branch doesn't conform to our new branch naming conventions YOU'RE FIRED

**Comment by tilt** [comment]
@manwithtwowatches I didn't take part in the meeting, so I'm not involved

File: app/assets/javascripts/angular/directives/work_packages/sort-header-directive.js [hunk 1]
```diff
@@ -58,21 +59,12 @@ angular.module('openproject.workPackages.directives')
...
-      };
-
       function setFullTitle() {
         if(!scope.sortable) scope.fullTitle = '';
 
         if(scope.currentSortDirection) {
           var sortDirectionText = (scope.currentSortDirection == 'asc') ? I18n.t('js.label_ascending') : I18n.t('js.label_descending');
           scope.fullTitle = sortDirectionText + " " + I18n.t('js.label_sorted_by') + ' \"' + scope.headerTitle + '\"';
-        } else {
-          scope.fullTitle = (I18n.t('js.label_sort_by') + ' \"' + scope.headerTitle + '\"');
```
Comment (manwithtwowatches) [inline app/assets/javascripts/angular/directives/work_packages/sort-header-directive.js:75]: @tilt was there a reason for removing the "sort by ..." title? is it required for the accessibility or anything?

Comment (tilt) [inline app/assets/javascripts/angular/directives/work_packages/sort-header-directive.js:75]: you cannot sort by clicking any more, it should rather display a 'label_open_dropdown' now

**Comment by ulferts** [comment]
<pre>
rspec ./spec/features/accessibility/work_packages/work_package_query_spec.rb
</pre>

fails

**Comment by tilt** [comment]
@ulferts Thanks, this was planned ;) tests will have to be fixed tomorrow

**Comment by myabc** [comment]
not tonight?

**Comment by manwithtwowatches** [comment]
@ulferts i'm fixing the specs just now

**Comment by myabc** [comment]
I rebased to make sure `Gemfile.lock` is gone for good.

**Pull request merged**